assembler/bsc_assembler.go

```diff
@@ -138,6 +142,22 @@ func (a *BSCAssembler) process(channelId types.ChannelId) error {
 	}
 	logging.Logger.Debugf("start seq and end enq are %d and %d", startSeq, endSequence)
 
+	a.alertSetMutex.Lock()
```

Comment:
Do we really need this `mutex` as I did not see any concurrent call of `process`

Reply:
Yeah its is unnecessary。